src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathMapperTest.java

```diff
@@ -761,7 +768,7 @@ public void testLastImport() throws Exception {
                 .getNextSibling()
                 .getNextSibling();
         final DetailAST[] expected = {expectedVariableDefNode};
-        assertThat("Result nodes differ from expected", actual, equalTo(expected));
+        assertWithMessage("Result nodes differ from expected").that(actual).isEqualTo(expected);
```

Comment:
Why not line wrap each method call? I think this improves readability:

```
        assertWithMessage("Result nodes differ from expected")
                .that(actual)
                 .isEqualTo(expected);

```

Reply:
Not against it, just saying for something this custom if we don't have a check for it, then it will get lost or not always done.

Reply:
Just so its clear, I won't be doing this as part of this effort. I am just working on a doing a straight conversion for now.

Reply:
In that case, resolved :)